assets/scss/main.scss

```diff
@@ -0,0 +1,5 @@
+@import 'abstract/fonts';
+@import 'abstract/functions';
+@import 'abstract/variables';
+@import 'base/typography';
+@import 'layout/navbarBanner';
```

Comment:
u layout ide nesto sto bi se trebalo ponavljati across web. u ovom slucaju bolje je da se to stavi u components